test/integration/blocks/verify/verify.js

```diff
@@ -922,7 +922,7 @@ describe('blocks/verify', () => {
 					expect(block1.totalFee.isEqualTo('0')).to.be.true;
 					expect(block1.totalAmount.isEqualTo('0')).to.be.true;
 					expect(block1.payloadLength).to.equal(0);
-					expect(block1.transactions).to.deep.eql([]);
+					expect(block1.transactions).to.deep.equal([]);
```

Comment:
@diego-G what is the reason behind changing the `to.deep.eql` to `to.deep.equal` apart from explicit readability?
if there is any other reason I would be glad to know. And if we are changing `to.deep.eql` to `to.deep.equal` why not change it in other places where it is `required` like `to.eql` to `to.equal` consistently?

Reply:
It was @shuse2's suggestion. I agreed because the majority of the cases are as `to.deep.equal`. We should define these rules in the developer's guide and apply them all at once.

Reply:
Ok, we can create an issue to change the same across the test suite. I am gonna approve the PR for now.

Reply:
https://github.com/LiskHQ/lisk/issues/2532